README.md

```diff
@@ -2,118 +2,192 @@
+  - [Run a `cql-proxy` docker image](https://github.com/datastax/cql-proxy#run-a-cql-proxy-docker-image)
+  - [Use Kubernetes](https://github.com/datastax/cql-proxy#use-kubernetes)
+
+
+## What is `cql-proxy`?
 
 ![cql-proxy](cql-proxy.png)
 
-## Getting started
+`cql-proxy` is designed to forward your application's CQL traffic to an appropriate database service. It listens on a local address and securely forwards that traffic.
 
-```sh
-go build
-```
+**Warning**: `cql-proxy` in its current state works well, and you should give it a try. However, it is still under development, so things might break or change. 
 
-Run against a [DataStax Astra][astra] cluster:
+Please give it a try and let us know what you think!
+## When to use `cql-proxy`
+
+The `cql-proxy` sidecar enables unsupported CQL drivers to work with [DataStax Astra][astra]. These drivers include both legacy DataStax [drivers] and community-maintained CQL drivers, such as the [gocql] driver and the [rust-driver].
+
+`cql-proxy` also enables applications that are currently using [Apache Cassandra][cassandra] or [DataStax Enterprise (DSE)][dse] to use Astra without requiring any code changes.  Your application just needs to be configured to use the proxy.
+
+If you're building a new application using DataStax [drivers], `cql-proxy` is not required, as the drivers can communicate directly with Astra. DataStax drivers have excellent support for Astra out-of-the-box, and are well-documented in the [driver-guide] guide. 
+
+## Configuration
+
+Use the `-h` or `--help` flag to display a listing all flags and their corresponding descriptions and environment variables (shown below as items starting with `$`):
 
 ```sh
-./cql-proxy --bundle <your-secure-connect-zip> \
-  --username <astra-client-id> --password <astra-client-secret>
+$ ./cql-proxy -h
+Usage: cql-proxy
+
+Flags:
+  -h, --help               Show context-sensitive help.
+  -b, --bundle=STRING      Path to secure connect bundle ($BUNDLE)
+  -u, --username=STRING    Username to use for authentication ($USERNAME)
+  -p, --password=STRING    Password to use for authentication ($PASSWORD)
+  -c, --contact-points=CONTACT-POINTS,...
+                           Contact points for cluster. Ignored if using the bundle path
+                           option ($CONTACT_POINTS).
+  -a, --bind=STRING        Address to use to bind serve ($BIND)
+      --debug              Show debug logging ($DEBUG)
+      --profiling          Enable profiling ($PROFILING)
 ```
 
-or using Docker as
+To pass configuration to `cql-proxy`, either command-line flags or environment variables can be used. Using the `docker` method as an example, the follwing samples show how username, password and bundle are defined with each method.
+### Using flags
 
 ```sh
 docker run -v <your-secure-connect-bundle.zip>:/tmp/scb.zip -p 9042:9042 \
-  --rm datastax/cql-proxy:v0.0.4 \
+  --rm datastax/cql-proxy:v0.0.2 \
```

Comment:
This should be the latest version `0.0.4` instead of `0.0.2`.

Reply:
Doing this for a readme in md actually seems to require pre-processing: https://stackoverflow.com/questions/54740161/use-variable-in-markdown-to-represent-url-base

If you were open to changing the readme to asciidoc, then such variable substitutions are easy. I created a readme.adoc here, if you want to see: https://github.com/polandll/antora-testing/tree/test_readme.